README.md

```diff
@@ -105,6 +122,22 @@ Distributed optimization using a shared database is also possible (see the corre
 python train.py --algo ppo --env MountainCar-v0 -optimize --study-name test --storage sqlite:///example.db
 ```
 
+### Default agent hyperparameters
+
+Note that the default hyperparameters used in the zoo when tuning are not always the same as the defaults provided in [stable-baselines3](https://stable-baselines3.readthedocs.io/en/master/modules/base.html). Consult the latest source code to be sure of these settings. For example:
+
+- PPO tuning assumes a network architecture with `ortho_init = False` when tuning, though it is `True` by [default](https://stable-baselines3.readthedocs.io/en/master/modules/ppo.html#ppo-policies).
+
+- PPO and A2C will use seperate architectures for policy and value networks, e.g. `net_arch=[dict(pi=[64, 64], vf=[64, 64])`, while the default is a shared network, `net_arch=[64, 64]`
```

Comment:
where did you see that the default was a shared network?

Reply:
Good question, I was basing that on 
https://github.com/DLR-RM/stable-baselines3/blob/944dfdafe496490cb6a7dc0e2198c576e4d2ba98/stable_baselines3/dqn/policies.py#L41-L42

but realize I may have been misreading how far this default is inherited or something and should have at least flagged this comment more as a supposition on my part than a firm assertion!  (As mentioned earlier, overall I've found it rather tricky to figure out what the defaults are, hence my attempt to document them more explicitly somewhere, though perhaps the README isn't the best place to state some of these.  All comments in the PR here are based on my notes from my poor attempts to read the source or experiments over the past month and may well be wrong, so I can't say how much I appreciate the chance to verify or correct these assumptions with you!)

Reply:
oh i see, you looked at DQN (which has only one Q-network) whereas PPO/A2C have two networks (actor and critic)

>tricky to figure out what the defaults

we should probably make it explicit in SB3 doc. but for short: we use the network architectures from the original papers (so [256, 256] for SAC, separated [64, 64] networks for PPO, [400, 300] for TD3, ...) and same for the default hyperparameters.